lib/lab_manager/sidekiq/boot.rb

```diff
@@ -1,4 +1,7 @@
 $LOAD_PATH << 'lib'
+require 'sidekiq'
 require 'lab_manager'
 
 LabManager.setup
+LabManager.logger = Sidekiq.logger
```

Comment:
Logger is set also in LabManager.setup. It would be better if all of logger initialization was in one place.